liquibase-core/src/main/java/liquibase/command/CommandArgumentNames.java

```diff
@@ -0,0 +1,11 @@
+package liquibase.command;
+
+/**
+ * A common place to store commonly used command argument names.
+ */
+public class CommandArgumentNames {
```

Comment:
This is maybe better as an enum?

Also maybe better named "CommonArgumentNames" vs. CommandArgumentNames"?

Reply:
Yes and yes. Resolved.